config/main.py

```diff
@@ -1045,20 +1138,98 @@ def add(session_name, src_ip, dst_ip, dscp, ttl, gre_type, queue, policer):
 
     if queue is not None:
         session_info['queue'] = queue
-    
+
+    if src_port is not None:
+        if get_interface_naming_mode() == "alias":
+            src_port_list = []
+            for port in src_port.split(","):
+                src_port_list.append(interface_alias_to_name(port))
+            src_port=",".join(src_port_list)
+
+        session_info['src_port'] = src_port
+        if direction is None:
+            direction = "both"
+
+    if direction is not None:
```

Comment:
Is this still needed with the default logic above?

Reply:
Removed the check.